source/includes/getting-started/_install.html.md

```diff
@@ -0,0 +1,265 @@
+3. If you *do not* have Chocolatey, Homebrew, or Anaconda installed on your system continue to follow our **Manual Install** intructions below.
+
+<aside class="notice">
+The two tabs to the right allow you to follow this guide as a Mac or Windows user. Instructions also vary slightly between Python 2 and Python 3 so please be sure to follow your version's instructions.
+</aside>
+
+## Manual Install
+
+### Windows
+
+* **Git**: Download and install Git from [Git for Windows][11]. 
+* **Python**: Download and install Python (3.6 or 2.7) from [python.org][3].
+
+<aside class="warning">
+For users installing <b>Python 2.7</b>: Make sure to include adding python to your PATH when customizing your install on the third screen. <br>
+For users installing <b>Python 3.6</b>: Make sure to check the box on the first screen of the install to add python to your PATH.
+</aside>
+
+* **Pip**: Python version 2.7.9 and greater come with pip, but if for some reason you need to use a different version of python please follow [these instructions][12].
+
+<aside class="notice">
+Please remember to reboot your computer after installing Python and Git to make sure all install settings take effect.
+</aside>
+
+**Great!** Now that you have Git, Python, and Pip installed, you can skip down to [Setup the Bonsai CLI][2]. 
+
+### Mac
+
+* **Git**: Git is more than likely already installed on your computer if you have Xcode, but if not, run `git` from your command prompt.
+* **Python**: Download and install Python (3.6 or 2.7) from [python.org][3].
+
+<aside class="notice">
+We highly recommend Mac users download Python directly so you can follow this guide on user-land Python rather than <a href="https://github.com/MacPython/wiki/wiki/Which-Python">system Python</a>.
+</aside>
+
+* **Pip**: Python version 2.7.9 and greater come with pip (or pip3 for version 3+), but if for some reason you need to use a different version of python please follow [these instructions][12].
+
+**Great!** Now that you have Git, Python, and Pip installed, you can skip down to [Setup the Bonsai CLI][2]. 
+
+## Install with Chocolatey or Homebrew
+
+If you *do not* have Chocolatey or Homebrew installed on your system, please skip this section.
+
+> Python 2
+
+```powershell
+choco install git
+choco install python2
+```
+```shell
+brew install python
+```
+
+> Python 3
+
+```powershell
+choco install git
+choco install python
+```
+```shell
+brew install python3
+```
+
+### Windows
+
+**If you're already familiar with [Chocolatey][14], you can follow the commands shown to the right instead of installing manually.**
+
+<aside class="notice">
+Please remember to run choco commands in administrator powershell/command prompt.
+</aside>
+
+### Mac
+
+**If you're already familiar with [Homebrew][6], you can follow the commands shown to the right instead of installing manually.**
```

Comment:
"instead of installing manually" is confusing to me. Are you refering to the manual install instructions above? I'm already in this section seems weird to mention it. I would just delete this phrase.